ofono/bridge.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+from time import sleep
+
+import pydbus
+from dbus.mainloop.glib import DBusGMainLoop
+from gi.repository import GLib
+
+from helpers import Singleton, setup_logger
+
+DBusGMainLoop(set_as_default=True)  # has to be called first
+
+logger = setup_logger(__name__, 'debug')
+logging.basicConfig(level=logging.DEBUG)
+
+
+class OfonoBridge(object):
+    """
+    Generic util class to bridge between ZPUI and ofono backend through D-Bus
+    """
+
+    def __init__(self):
+        super(OfonoBridge, self).__init__()
+
+        bus = pydbus.SystemBus()
+        manager = bus.get('org.ofono', '/')
+        modem_path = manager.GetModems()[0][0]
+
+        if modem_path != '/sim900_0':
```

Comment:
It's a check because if you run ofono with some plugins, you might have a `/stk_test` virtual modem (standing for `SimToolKit test`). I wanted to be able to check the modem was correctly initialized at this point, but it might be an issue when we switch to SIM52XX later on. Shall I remove it ?

Reply:
"check if it's correctly initialized" - you mean, check if it's available? If so, I'm guessing it might still be there in some cases, just not the 0th element in the list of the modems that you'll get. I suggest using `if "/sim900_0" in manager.getModems()[0]` or something along those lines.